skydive-grafana/docker/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM debian:jessie
```

Comment:
@etsauer Yes, I could move the Dockerfile to the root directory, not a problem but I'm trying to use the following directory layout when I create any resource for OCP:
/repo (repository root directory)
/repo/kube (kubernetes objects in JSON format)
/repo/docker (Dockerfile to create the image. Sub directories per image)
/repo/docker/src (source files required to build the image)
/repo/img (png images used for documentation)

Is just a schema I'm following in order to keep the repo more "clean". It'd not make sense for a simple repo, but it could be useful for more complex repositories with more than one image. Any thoughts/comments on this?

@lpsantil yes, we could move this to CentOS/Fedora, I used Debian because it's the base image Grafana is using officially (https://github.com/grafana/grafana-docker/blob/master/Dockerfile) but it should not be a problem to move this to a RHEL based image. Will work on that if you guys think this make more sense